README.md

```diff
@@ -29,6 +29,11 @@ You can run the dashboard locally by following these steps:
 ### Configuring Dashboard Environment
 CCSM CDS Dashboard relies on environment variables for configuration. Configuration options are described in the [Environment Setting for Dashboard](https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/wiki/Environment-Setting-for-Dashboard) section of the Wiki.
 
+### Environment Variables to be set
```

Comment:
Awesome. The text look great, nothing to change there. My only concern is that having these two environment variables described here, and then the rest of the environment variables described in the [Wiki](https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/wiki/Environment-Setting-for-Dashboard) is a bit confusing. The section just above this called "Configuring Dashboard Environment" says that configuration options are described in the Wiki link. But then this section appears to say that the configuration options are just these two env variables listed here. I think the two sections where environment variables are described should be combined. I'm open to the environment variables being described either in the Wiki or here in the README, but I think it would be nice to keep them all together. Maybe we can add these two here to the table in the Wiki article, and then remove them here. Thoughts?

Reply:
Moved to wiki page